README.md

```diff
@@ -57,6 +57,24 @@ The tooling has evolved over time and therefore when generating new packages, yo
 This is because the new package has a dependency on an existing package, it was regenerated and changes were detected from
 when it was originally generated.
 
+#### Backends
+The tooling support two backends for generating reference assemblies: [the Cci-based GenAPI](https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.GenAPI) and [the Roslyn-based GenAPI](https://github.com/dotnet/sdk/tree/main/src/GenAPI). The default one is Roslyn-based. You can switch between backends using the `--genapi-backend` parameter that supports `cci` and `roslyn` values.
+
+```bash
+./generate.sh --pkg system.buffers,4.5.1 --genapi-backend roslyn --dest <destination path>
```

Comment:
What is the intent of this sample?  I am assuming it is to show the use of the backend option.  This sample doesn't make the most sense if roslyn is the default backend.   Also the use of --dest is not necessary in the canonical examples.